src/pykeen/nn/modules.py

```diff
@@ -652,7 +652,7 @@ def _prepare_hrt_for_functional(
         r: RelationRepresentation,
         t: TailRepresentation,
     ) -> MutableMapping[str, torch.FloatTensor]:  # noqa: D102
-        return dict(h=h, r=r, t=t[0], t_bias=t[1])
+        return dict(h=h[0], r=r, t=t[0], t_bias=t[1])
```

Comment:
here, we select entity biases for the head, although they are never used. This should be adapted in the base model, but becomes apparent here, since ConvE is the only model with tail biases (afaik).

Reply:
Wait there’s at least one other model that has head and tail specific representations

Reply:
`CP` uses separate head / tail representations.